packages/diagram-ui/src/features/viewport/setModelActionHandler.ts

```diff
@@ -8,6 +8,6 @@ import { Action, FitToScreenAction } from "sprotty-protocol";
 @injectable()
 export class SetModelActionHandler implements IActionHandler {
     handle(_: Action): ICommand | Action | void {
-        return FitToScreenAction.create([]);
+        return FitToScreenAction.create([], { padding: 50 });
```

Comment:
I would prefer also having a maxZoom (or however the key is called exactly) here
With a large monitor, you now still have a potentially giant class (but with padding now)
I think a maxZoom of 2 or so should be reasonable (may need some testing)

Reply:
Funny thing: I don't see any difference when setting this setting.
Neither for large nor for small diagrams.

Reply:
What exactly is it supposed to do?

Reply:
I hoped it would limit the zoom

Reply:
Doesn't look like that's the case, or I'm doing something wrong:
![image](https://github.com/user-attachments/assets/9d11f2ef-46e1-41e3-9b1c-65224a3b3d42)

Reply:
not in that direction
when you have a very small class, it should not fill the whole screen
![image](https://github.com/user-attachments/assets/7ee67ef9-2a54-4171-bedf-59b8b651024c)
for me, it works fine

Reply:
(now this class does not almost fill the right side anymore, but has only a zoom of 2)